src/caffeine/caffeine.c

```diff
@@ -90,6 +90,15 @@ void caf_deallocate(mspace heap, void* mem)
   mspace_free(heap, mem);
 }
 
+// take address in a segment and convert to an address on given image
+void* caf_convert_base_addr(void* addr, int image, gex_TM_t initial_team)
```

Comment:
With the change suggested above, the `initial_team` argument here can and should be removed.